.github/workflows/publish-cuda-docker-image.yml

```
@@ -66,7 +66,7 @@ jobs:
           # See https://github.com/docker/metadata-action#tags-input for the
           # format this should take.
           # DUE TO JSON LIMITATIONS, NEWLINES MUST BE EXPLICITLY ADDED AS \n
-          additional_image_tags: 'type=raw,value=${{ inputs.docker_tag }}'
+          additional_image_tags: ${{ inputs.docker_tag && format('type=raw,value={0}', inputs.docker_tag) || ''}}
```

Similar question like in the other repo. Shouldn't this append `-cuda`?
```suggestion
          additional_image_tags: ${{ inputs.docker_tag && format('type=raw,value={0}', inputs.docker_tag)-cuda || ''}}
```

---

Same as in the other comment.